### PR TITLE
EES-2937 - fix release_content.robot

### PR DESCRIPTION
**Pull request by luke-h1**
image

### CODE REVIEW

File: tests/robot-tests/tests/admin/bau/release_content.robot [hunk 1]
```diff
@@ -79,7 +80,7 @@ Add Useful information related page link to release
     user clicks button    Create link
 
 Add secondary statistics
-    user waits until page does not contain element    id:${SECONDARY_STATS_TABLE_TAB_ID}
+    user waits until page does not contain element    //*[@id="${SECONDARY_STATS_TABLE_TAB_ID}"]
```
Comment (ntsim) [inline tests/robot-tests/tests/admin/bau/release_content.robot:83]: Wut - why is this working but `id:` isn't?

Comment (luke-h1) [reply tests/robot-tests/tests/admin/bau/release_content.robot:83]: Not sure, I can remember something about the custom selector being flaky in some instances

Comment (ntsim) [reply tests/robot-tests/tests/admin/bau/release_content.robot:83]: Yeh, but I mean this just doesn't make any sense, are you sure it's actually failing on this?

Comment (luke-h1) [reply tests/robot-tests/tests/admin/bau/release_content.robot:83]: Yeah, it's strange. I'm pretty sure this is the cause of it though. When using the `id:` selector, selenium consistently throws an `element not found in x amount of seconds` error

Comment (ntsim) [reply tests/robot-tests/tests/admin/bau/release_content.robot:83]: Cool, yeh fair enough! Happy to approve this, but would be good to look into this further to figure out if there's something else going on here

**Review by ntsim — APPROVED** [review]

File: tests/robot-tests/tests/admin/bau/release_content.robot (hunk 1 above)
Comment (luke-h1) [reply tests/robot-tests/tests/admin/bau/release_content.robot:83]: ta, sounds good to me. I'll put it on my todo list

**Pull request merged**